ci/README.md

```diff
@@ -0,0 +1,7 @@
+`fly -t eq login -c CONCOURSE_URL`
```

Comment:
The concourse team is `main`, we should either update the README, or change the concourse team to `eq`